docker/hisat2/Dockerfile

```diff
@@ -10,7 +10,25 @@ WORKDIR /opt/tools
 
 RUN \
   apt update && \ 
-  apt install -y liblzma-dev libbz2-dev cmake automake curl libboost-all-dev wget build-essential gcc-multilib zlib1g-dev libxml2-dev libncurses5-dev zip unzip samtools
+  apt install -y \
+	liblzma-dev \
+	libbz2-dev \
+	cmake \
+	automake \
+	curl \
+	libboost-all-dev \
+	wget \
+	build-essential \	
+	gcc-multilib \
+	zlib1g-dev \
+	libxml2-dev \
+	libncurses5-dev \	
+	zip unzip \
+	samtools \
+	git \
+	r-base \
+	r-base-core \
+    r-base-dev
```

Comment:
Missing some spaces on line 31 to have it line up, and best practices would have these in alphabetical order.